bipen/src/main/java/com/lftechnology/phpjava/assignmenttwo/Multiplier.java

```diff
@@ -0,0 +1,62 @@
+package com.lftechnology.phpjava.assignmenttwo;
+
+import java.util.Scanner;
+
+/**
+ * Class to generate the sum of all the multiples of two numbers below certain multiplier.
+ * 
+ * @author bipen
+ *
+ */
+public class Multiplier {
+
+    private Scanner scanner;
+
+    /**
+     * Main method that runs the program and generated the sum of all the multiples of two numbers below certain multiplier.
+     * 
+     * @author bipen
+     */
+    public void run() {
+        this.scanner = new Scanner(System.in);
+        System.out.println("Enter Multiplier:");
+        this.validateInput();
+        Integer multiplier = this.scanner.nextInt();
+        System.out.println("Enter First Number:");
+        this.validateInput();
+        Integer firstNumber = this.scanner.nextInt();
+        System.out.println("Enter Second Number:");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS106)